onwuvic/MyDiary
# #158909055 Create user sign-up page

#### What does this PR do?
A User sign-up page interface

#### Description of Task to be completed?
As a User, I should be able to create an account on the application so that I can log in, then start writing their diaries/entries

#### How should this be manually tested?
Given A user
When User navigates to the sign-up page
Then User should see a simple and descriptive form with First Name, Last Name, Email, Password and Confirm Password fields

#### What are the relevant pivotal tracker stories?
#158909055
